web/__init__.py

```diff
@@ -16,15 +14,160 @@
     webapi,
     wsgi,
 )
-from .application import *
-from .db import *
-from .debugerror import *
-from .http import *
-from .httpserver import *
-from .net import *
-from .utils import *
-from .webapi import *
-from .wsgi import *
+from .application import (  # noqa: F401
```

Comment:
Please add some (brief) comment lines in `web/__init__.py` to mention why we do this change, and emphasize not to introduce any new functions in the `from .XX import (...)`. For example:

> In web.py-0.40 and earlier releases, we use `from .XX import *` to import all
> (exported) functions from sub-modules to `web.*` namespace. For example,
> `web.reparam()` (should be `web.db.reparam()`), `web.safestr()` (should be
> `web.utils.safestr()`). We decide to stop this bad practice and encourage
> users to use clear namespaces (use `web.XX.YY` instead of `web.YY`).
>
> These explicit imports are kept for backward-compatibility, do not import
> any namespace under `web.` for new functions.
>
> FYI: https://www.asmeurer.com/removestar/

Just an example, please update/adjust with better / clearer explanation.